packages/botfuel-dialog/src/config.js

```diff
@@ -26,6 +26,7 @@ const defaultConfig = {
   brain: 'memory',
   intentThreshold: 0.8,
   logger: 'info',
+  nlu: 'botfuel',
```

Comment:
wondering if we won't need something a bit more flexible 
nlu: { name: 'botfuel' }
what do you think?

Reply:
👍  This is useful to pass config to the NLU module.

Reply:
Then `intentThreshold` should be in that array:

```
const defaultConfig = {
  path: process.cwd(),
  locale: 'en',
  adapter: 'shell',
  brain: 'memory',
  logger: 'info',
  nlu: {
    name: 'botfuel',
    intentThreshold: 0.8,
  },
  multiIntent: false,
  conversationDuration: 86400000, // in ms
};
```

...and that's a breaking change.

Reply:
If we do that, I would also suggest to allow config for brains and adapters:

```
const defaultConfig = {
  path: process.cwd(),
  locale: 'en',
  adapter: {
    name: 'shell',
  },
  brain: {
    name: 'memory',
  },
  logger: 'info',
  nlu: {
    name: 'botfuel',
    intentThreshold: 0.8,
  },
  multiIntent: false,
  conversationDuration: 86400000, // in ms
};

Reply:
Make that change.